core/src/main/java/com/linecorp/armeria/server/ServiceConfig.java

```diff
@@ -184,12 +188,24 @@ public HttpService service() {
      *       {@code com.foo.ThriftService$Iface})</li>
      *   <li>{@link HttpService} and annotated service - an innermost class name</li>
      * </ul>
+     *
+     * @deprecated Use {@link #defaultServiceNaming} instead.
```

Comment:
```suggestion
     * @deprecated Use {@link #defaultServiceNaming()} instead.
```

Reply:
Thanks again for precise review. Updated.